### DISCUSSION TITLE
Need clarity on Notice Publication Date Preferred business rule

### DISCUSSION THREAD

**Discussion by ghost** [OP-TED/eForms-SDK#296] [Q&A] [answered]
Hi,

Need clarification on business rule for Notice Publication Date Preferred. Constraint is defined as below - 

Co-constraint in EFX
((BT-738-notice - BT-05(a)-notice) < P92D) and (BT-738-notice > BT-05(a)-notice)
Applies if Notice Publication Date Preferred (BT-738) exists.

Condition in EFX
BT-738-notice is present

<img width="829" alt="Screenshot 2023-01-09 at 2 53 10 PM" src="https://user-images.githubusercontent.com/122265333/211275950-9f769ba2-284a-49eb-b4aa-5e7366f72923.png">

What does P92D means here? Let me know if more information is required.

Thanks,
Chandresh Parmar

**Comment by YvesJo** [comment, marked as answer]
Hi,
P92D designates a duration of 92 days. Therefore the test checks that, when specified, the Notice Publication Date Preferred is within 92 days after the Notice Dispatch Date.
KR,

**Reply by ghost** [reply]
Thanks @YvesJo for quick response.
